<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <title>Test Report</title>
    <link href="assets/style.css" rel="stylesheet" type="text/css"/></head>
  <body onLoad="init()">
    <script>/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */


function toArray(iter) {
    if (iter === null) {
        return null;
    }
    return Array.prototype.slice.call(iter);
}

function find(selector, elem) { // eslint-disable-line no-redeclare
    if (!elem) {
        elem = document;
    }
    return elem.querySelector(selector);
}

function findAll(selector, elem) {
    if (!elem) {
        elem = document;
    }
    return toArray(elem.querySelectorAll(selector));
}

function sortColumn(elem) {
    toggleSortStates(elem);
    const colIndex = toArray(elem.parentNode.childNodes).indexOf(elem);
    let key;
    if (elem.classList.contains('result')) {
        key = keyResult;
    } else if (elem.classList.contains('links')) {
        key = keyLink;
    } else {
        key = keyAlpha;
    }
    sortTable(elem, key(colIndex));
}

function showAllExtras() { // eslint-disable-line no-unused-vars
    findAll('.col-result').forEach(showExtras);
}

function hideAllExtras() { // eslint-disable-line no-unused-vars
    findAll('.col-result').forEach(hideExtras);
}

function showExtras(colresultElem) {
    const extras = colresultElem.parentNode.nextElementSibling;
    const expandcollapse = colresultElem.firstElementChild;
    extras.classList.remove('collapsed');
    expandcollapse.classList.remove('expander');
    expandcollapse.classList.add('collapser');
}

function hideExtras(colresultElem) {
    const extras = colresultElem.parentNode.nextElementSibling;
    const expandcollapse = colresultElem.firstElementChild;
    extras.classList.add('collapsed');
    expandcollapse.classList.remove('collapser');
    expandcollapse.classList.add('expander');
}

function showFilters() {
    const filterItems = document.getElementsByClassName('filter');
    for (let i = 0; i < filterItems.length; i++)
        filterItems[i].hidden = false;
}

function addCollapse() {
    // Add links for show/hide all
    const resulttable = find('table#results-table');
    const showhideall = document.createElement('p');
    showhideall.innerHTML = '<a href="javascript:showAllExtras()">Show all details</a> / ' +
                            '<a href="javascript:hideAllExtras()">Hide all details</a>';
    resulttable.parentElement.insertBefore(showhideall, resulttable);

    // Add show/hide link to each result
    findAll('.col-result').forEach(function(elem) {
        const collapsed = getQueryParameter('collapsed') || 'Passed';
        const extras = elem.parentNode.nextElementSibling;
        const expandcollapse = document.createElement('span');
        if (extras.classList.contains('collapsed')) {
            expandcollapse.classList.add('expander');
        } else if (collapsed.includes(elem.innerHTML)) {
            extras.classList.add('collapsed');
            expandcollapse.classList.add('expander');
        } else {
            expandcollapse.classList.add('collapser');
        }
        elem.appendChild(expandcollapse);

        elem.addEventListener('click', function(event) {
            if (event.currentTarget.parentNode.nextElementSibling.classList.contains('collapsed')) {
                showExtras(event.currentTarget);
            } else {
                hideExtras(event.currentTarget);
            }
        });
    });
}

function getQueryParameter(name) {
    const match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
    return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
}

function init () { // eslint-disable-line no-unused-vars
    resetSortHeaders();

    addCollapse();

    showFilters();

    sortColumn(find('.initial-sort'));

    findAll('.sortable').forEach(function(elem) {
        elem.addEventListener('click',
            function() {
                sortColumn(elem);
            }, false);
    });
}

function sortTable(clicked, keyFunc) {
    const rows = findAll('.results-table-row');
    const reversed = !clicked.classList.contains('asc');
    const sortedRows = sort(rows, keyFunc, reversed);
    /* Whole table is removed here because browsers acts much slower
     * when appending existing elements.
     */
    const thead = document.getElementById('results-table-head');
    document.getElementById('results-table').remove();
    const parent = document.createElement('table');
    parent.id = 'results-table';
    parent.appendChild(thead);
    sortedRows.forEach(function(elem) {
        parent.appendChild(elem);
    });
    document.getElementsByTagName('BODY')[0].appendChild(parent);
}

function sort(items, keyFunc, reversed) {
    const sortArray = items.map(function(item, i) {
        return [keyFunc(item), i];
    });

    sortArray.sort(function(a, b) {
        const keyA = a[0];
        const keyB = b[0];

        if (keyA == keyB) return 0;

        if (reversed) {
            return keyA < keyB ? 1 : -1;
        } else {
            return keyA > keyB ? 1 : -1;
        }
    });

    return sortArray.map(function(item) {
        const index = item[1];
        return items[index];
    });
}

function keyAlpha(colIndex) {
    return function(elem) {
        return elem.childNodes[1].childNodes[colIndex].firstChild.data.toLowerCase();
    };
}

function keyLink(colIndex) {
    return function(elem) {
        const dataCell = elem.childNodes[1].childNodes[colIndex].firstChild;
        return dataCell == null ? '' : dataCell.innerText.toLowerCase();
    };
}

function keyResult(colIndex) {
    return function(elem) {
        const strings = ['Error', 'Failed', 'Rerun', 'XFailed', 'XPassed',
            'Skipped', 'Passed'];
        return strings.indexOf(elem.childNodes[1].childNodes[colIndex].firstChild.data);
    };
}

function resetSortHeaders() {
    findAll('.sort-icon').forEach(function(elem) {
        elem.parentNode.removeChild(elem);
    });
    findAll('.sortable').forEach(function(elem) {
        const icon = document.createElement('div');
        icon.className = 'sort-icon';
        icon.textContent = 'vvv';
        elem.insertBefore(icon, elem.firstChild);
        elem.classList.remove('desc', 'active');
        elem.classList.add('asc', 'inactive');
    });
}

function toggleSortStates(elem) {
    //if active, toggle between asc and desc
    if (elem.classList.contains('active')) {
        elem.classList.toggle('asc');
        elem.classList.toggle('desc');
    }

    //if inactive, reset all other functions and add ascending active
    if (elem.classList.contains('inactive')) {
        resetSortHeaders();
        elem.classList.remove('inactive');
        elem.classList.add('active');
    }
}

function isAllRowsHidden(value) {
    return value.hidden == false;
}

function filterTable(elem) { // eslint-disable-line no-unused-vars
    const outcomeAtt = 'data-test-result';
    const outcome = elem.getAttribute(outcomeAtt);
    const classOutcome = outcome + ' results-table-row';
    const outcomeRows = document.getElementsByClassName(classOutcome);

    for(let i = 0; i < outcomeRows.length; i++){
        outcomeRows[i].hidden = !elem.checked;
    }

    const rows = findAll('.results-table-row').filter(isAllRowsHidden);
    const allRowsHidden = rows.length == 0 ? true : false;
    const notFoundMessage = document.getElementById('not-found-message');
    notFoundMessage.hidden = !allRowsHidden;
}
</script>
    <h1>r.html</h1>
    <p>Report generated on 07-May-2022 at 02:34:09 by <a href="https://pypi.python.org/pypi/pytest-html">pytest-html</a> v3.1.1</p>
    <h2>Environment</h2>
    <table id="environment">
      <tr>
        <td>Packages</td>
        <td>{"pluggy": "1.0.0", "py": "1.11.0", "pytest": "7.1.2"}</td></tr>
      <tr>
        <td>Platform</td>
        <td>Windows-10-10.0.19044-SP0</td></tr>
      <tr>
        <td>Plugins</td>
        <td>{"html": "3.1.1", "metadata": "2.0.1"}</td></tr>
      <tr>
        <td>Python</td>
        <td>3.10.4</td></tr></table>
    <h2>Summary</h2>
    <p>2 tests ran in 46.13 seconds. </p>
    <p class="filter" hidden="true">(Un)check the boxes to filter the results.</p><input checked="true" class="filter" data-test-result="passed" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="passed">2 passed</span>, <input checked="true" class="filter" data-test-result="skipped" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="skipped">0 skipped</span>, <input checked="true" class="filter" data-test-result="failed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="failed">0 failed</span>, <input checked="true" class="filter" data-test-result="error" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="error">0 errors</span>, <input checked="true" class="filter" data-test-result="xfailed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="xfailed">0 expected failures</span>, <input checked="true" class="filter" data-test-result="xpassed" disabled="true" hidden="true" name="filter_checkbox" onChange="filterTable(this)" type="checkbox"/><span class="xpassed">0 unexpected passes</span>
    <h2>Results</h2>
    <table id="results-table">
      <thead id="results-table-head">
        <tr>
          <th class="sortable result initial-sort" col="result">Result</th>
          <th class="sortable" col="name">Test</th>
          <th class="sortable" col="duration">Duration</th>
          <th class="sortable links" col="links">Links</th></tr>
        <tr hidden="true" id="not-found-message">
          <th colspan="4">No results found. Try to check the filters</th></tr></thead>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">testofHome.py::Test_Of_Home::test_firstPageCheck[GetData0]</td>
          <td class="col-duration">24.55</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> -----------------------------Captured stdout setup------------------------------ <br/>Name
{&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}
{&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}
{&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}
[{&#x27;name&#x27;: &#x27;Yuvi&#x27;, &#x27;email&#x27;: &#x27;yuvi@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Yuvi&#x27;, &#x27;email&#x27;: &#x27;yuvi@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}]
[{&#x27;name&#x27;: &#x27;Yuvi&#x27;, &#x27;email&#x27;: &#x27;yuvi@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Yuvi&#x27;, &#x27;email&#x27;: &#x27;yuvi@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}]
<br/> ------------------------------Captured stderr call------------------------------ <br/>--- Logging error ---
Traceback (most recent call last):
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 1100, in emit
    msg = self.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 943, in format
    return fmt.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 681, in format
    s = self.formatMessage(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 650, in formatMessage
    return self._style.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 440, in format
    return self._format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 436, in _format
    return self._fmt % values
ValueError: unsupported format character &#x27;)&#x27; (0x29) at index 51
Call stack:
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 86, in _run_code
    exec(code, run_globals)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\Scripts\py.test.exe\__main__.py&quot;, line 7, in &lt;module&gt;
    sys.exit(console_main())
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 187, in console_main
    code = main()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 164, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 315, in pytest_cmdline_main
    return wrap_session(config, _main)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 268, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 322, in _main
    config.hook.pytest_runtestloop(session=session)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 347, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 111, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 130, in runtestprotocol
    reports.append(call_and_report(item, &quot;call&quot;, log))
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 219, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 258, in call_runtest_hook
    return CallInfo.from_call(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 338, in from_call
    result: Optional[TResult] = func()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 259, in &lt;lambda&gt;
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 166, in pytest_runtest_call
    item.runtest()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 1761, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 192, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File &quot;C:\Users\Yuvraj verma\Desktop\SeleniumTesting\pytestfolder\HomeTest\testofHome.py&quot;, line 31, in test_firstPageCheck
    lg.info(&quot;Sending the name&quot;+GetData[&quot;name&quot;])
Message: &#x27;Sending the nameYuvi&#x27;
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 1100, in emit
    msg = self.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 943, in format
    return fmt.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 681, in format
    s = self.formatMessage(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 650, in formatMessage
    return self._style.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 440, in format
    return self._format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 436, in _format
    return self._fmt % values
ValueError: unsupported format character &#x27;)&#x27; (0x29) at index 51
Call stack:
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 86, in _run_code
    exec(code, run_globals)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\Scripts\py.test.exe\__main__.py&quot;, line 7, in &lt;module&gt;
    sys.exit(console_main())
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 187, in console_main
    code = main()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 164, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 315, in pytest_cmdline_main
    return wrap_session(config, _main)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 268, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 322, in _main
    config.hook.pytest_runtestloop(session=session)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 347, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 111, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 130, in runtestprotocol
    reports.append(call_and_report(item, &quot;call&quot;, log))
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 219, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 258, in call_runtest_hook
    return CallInfo.from_call(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 338, in from_call
    result: Optional[TResult] = func()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 259, in &lt;lambda&gt;
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 166, in pytest_runtest_call
    item.runtest()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 1761, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 192, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File &quot;C:\Users\Yuvraj verma\Desktop\SeleniumTesting\pytestfolder\HomeTest\testofHome.py&quot;, line 38, in test_firstPageCheck
    lg.info(&quot;Sending the email&quot;+GetData[&quot;email&quot;])
Message: &#x27;Sending the emailyuvi@gmail.com&#x27;
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 1100, in emit
    msg = self.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 943, in format
    return fmt.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 681, in format
    s = self.formatMessage(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 650, in formatMessage
    return self._style.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 440, in format
    return self._format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 436, in _format
    return self._fmt % values
ValueError: unsupported format character &#x27;)&#x27; (0x29) at index 51
Call stack:
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 86, in _run_code
    exec(code, run_globals)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\Scripts\py.test.exe\__main__.py&quot;, line 7, in &lt;module&gt;
    sys.exit(console_main())
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 187, in console_main
    code = main()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 164, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 315, in pytest_cmdline_main
    return wrap_session(config, _main)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 268, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 322, in _main
    config.hook.pytest_runtestloop(session=session)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 347, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 111, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 130, in runtestprotocol
    reports.append(call_and_report(item, &quot;call&quot;, log))
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 219, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 258, in call_runtest_hook
    return CallInfo.from_call(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 338, in from_call
    result: Optional[TResult] = func()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 259, in &lt;lambda&gt;
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 166, in pytest_runtest_call
    item.runtest()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 1761, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 192, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File &quot;C:\Users\Yuvraj verma\Desktop\SeleniumTesting\pytestfolder\HomeTest\testofHome.py&quot;, line 62, in test_firstPageCheck
    lg.info(&quot;Sending the gender&quot;+GetData[&quot;gender&quot;])
Message: &#x27;Sending the genderMale&#x27;
Arguments: ()
<br/> -------------------------------Captured log call-------------------------------- <br/>INFO     test_firstPageCheck:testofHome.py:31 Sending the nameYuvi
INFO     test_firstPageCheck:testofHome.py:38 Sending the emailyuvi@gmail.com
INFO     test_firstPageCheck:testofHome.py:62 Sending the genderMale<br/></div></td></tr></tbody>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">testofHome.py::Test_Of_Home::test_firstPageCheck[GetData1]</td>
          <td class="col-duration">21.22</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> -----------------------------Captured stdout setup------------------------------ <br/>Name
{&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}
{&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}
{&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}
[{&#x27;name&#x27;: &#x27;Yuvi&#x27;, &#x27;email&#x27;: &#x27;yuvi@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Yuvi&#x27;, &#x27;email&#x27;: &#x27;yuvi@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Yuvi&#x27;, &#x27;email&#x27;: &#x27;yuvi@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}]
[{&#x27;name&#x27;: &#x27;Yuvi&#x27;, &#x27;email&#x27;: &#x27;yuvi@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Yuvi&#x27;, &#x27;email&#x27;: &#x27;yuvi@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Yuvi&#x27;, &#x27;email&#x27;: &#x27;yuvi@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}, {&#x27;name&#x27;: &#x27;Tanger&#x27;, &#x27;email&#x27;: &#x27;Tanger@gmail.com&#x27;, &#x27;gender&#x27;: &#x27;Male&#x27;}]
<br/> ------------------------------Captured stderr call------------------------------ <br/>--- Logging error ---
Traceback (most recent call last):
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 1100, in emit
    msg = self.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 943, in format
    return fmt.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 681, in format
    s = self.formatMessage(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 650, in formatMessage
    return self._style.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 440, in format
    return self._format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 436, in _format
    return self._fmt % values
ValueError: unsupported format character &#x27;)&#x27; (0x29) at index 51
Call stack:
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 86, in _run_code
    exec(code, run_globals)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\Scripts\py.test.exe\__main__.py&quot;, line 7, in &lt;module&gt;
    sys.exit(console_main())
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 187, in console_main
    code = main()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 164, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 315, in pytest_cmdline_main
    return wrap_session(config, _main)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 268, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 322, in _main
    config.hook.pytest_runtestloop(session=session)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 347, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 111, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 130, in runtestprotocol
    reports.append(call_and_report(item, &quot;call&quot;, log))
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 219, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 258, in call_runtest_hook
    return CallInfo.from_call(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 338, in from_call
    result: Optional[TResult] = func()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 259, in &lt;lambda&gt;
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 166, in pytest_runtest_call
    item.runtest()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 1761, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 192, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File &quot;C:\Users\Yuvraj verma\Desktop\SeleniumTesting\pytestfolder\HomeTest\testofHome.py&quot;, line 31, in test_firstPageCheck
    lg.info(&quot;Sending the name&quot;+GetData[&quot;name&quot;])
Message: &#x27;Sending the nameTanger&#x27;
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 1100, in emit
    msg = self.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 943, in format
    return fmt.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 681, in format
    s = self.formatMessage(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 650, in formatMessage
    return self._style.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 440, in format
    return self._format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 436, in _format
    return self._fmt % values
ValueError: unsupported format character &#x27;)&#x27; (0x29) at index 51
Call stack:
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 86, in _run_code
    exec(code, run_globals)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\Scripts\py.test.exe\__main__.py&quot;, line 7, in &lt;module&gt;
    sys.exit(console_main())
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 187, in console_main
    code = main()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 164, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 315, in pytest_cmdline_main
    return wrap_session(config, _main)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 268, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 322, in _main
    config.hook.pytest_runtestloop(session=session)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 347, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 111, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 130, in runtestprotocol
    reports.append(call_and_report(item, &quot;call&quot;, log))
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 219, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 258, in call_runtest_hook
    return CallInfo.from_call(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 338, in from_call
    result: Optional[TResult] = func()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 259, in &lt;lambda&gt;
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 166, in pytest_runtest_call
    item.runtest()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 1761, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 192, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File &quot;C:\Users\Yuvraj verma\Desktop\SeleniumTesting\pytestfolder\HomeTest\testofHome.py&quot;, line 31, in test_firstPageCheck
    lg.info(&quot;Sending the name&quot;+GetData[&quot;name&quot;])
Message: &#x27;Sending the nameTanger&#x27;
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 1100, in emit
    msg = self.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 943, in format
    return fmt.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 681, in format
    s = self.formatMessage(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 650, in formatMessage
    return self._style.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 440, in format
    return self._format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 436, in _format
    return self._fmt % values
ValueError: unsupported format character &#x27;)&#x27; (0x29) at index 51
Call stack:
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 86, in _run_code
    exec(code, run_globals)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\Scripts\py.test.exe\__main__.py&quot;, line 7, in &lt;module&gt;
    sys.exit(console_main())
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 187, in console_main
    code = main()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 164, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 315, in pytest_cmdline_main
    return wrap_session(config, _main)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 268, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 322, in _main
    config.hook.pytest_runtestloop(session=session)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 347, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 111, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 130, in runtestprotocol
    reports.append(call_and_report(item, &quot;call&quot;, log))
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 219, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 258, in call_runtest_hook
    return CallInfo.from_call(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 338, in from_call
    result: Optional[TResult] = func()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 259, in &lt;lambda&gt;
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 166, in pytest_runtest_call
    item.runtest()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 1761, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 192, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File &quot;C:\Users\Yuvraj verma\Desktop\SeleniumTesting\pytestfolder\HomeTest\testofHome.py&quot;, line 38, in test_firstPageCheck
    lg.info(&quot;Sending the email&quot;+GetData[&quot;email&quot;])
Message: &#x27;Sending the emailTanger@gmail.com&#x27;
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 1100, in emit
    msg = self.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 943, in format
    return fmt.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 681, in format
    s = self.formatMessage(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 650, in formatMessage
    return self._style.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 440, in format
    return self._format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 436, in _format
    return self._fmt % values
ValueError: unsupported format character &#x27;)&#x27; (0x29) at index 51
Call stack:
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 86, in _run_code
    exec(code, run_globals)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\Scripts\py.test.exe\__main__.py&quot;, line 7, in &lt;module&gt;
    sys.exit(console_main())
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 187, in console_main
    code = main()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 164, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 315, in pytest_cmdline_main
    return wrap_session(config, _main)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 268, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 322, in _main
    config.hook.pytest_runtestloop(session=session)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 347, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 111, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 130, in runtestprotocol
    reports.append(call_and_report(item, &quot;call&quot;, log))
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 219, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 258, in call_runtest_hook
    return CallInfo.from_call(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 338, in from_call
    result: Optional[TResult] = func()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 259, in &lt;lambda&gt;
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 166, in pytest_runtest_call
    item.runtest()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 1761, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 192, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File &quot;C:\Users\Yuvraj verma\Desktop\SeleniumTesting\pytestfolder\HomeTest\testofHome.py&quot;, line 38, in test_firstPageCheck
    lg.info(&quot;Sending the email&quot;+GetData[&quot;email&quot;])
Message: &#x27;Sending the emailTanger@gmail.com&#x27;
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 1100, in emit
    msg = self.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 943, in format
    return fmt.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 681, in format
    s = self.formatMessage(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 650, in formatMessage
    return self._style.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 440, in format
    return self._format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 436, in _format
    return self._fmt % values
ValueError: unsupported format character &#x27;)&#x27; (0x29) at index 51
Call stack:
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 86, in _run_code
    exec(code, run_globals)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\Scripts\py.test.exe\__main__.py&quot;, line 7, in &lt;module&gt;
    sys.exit(console_main())
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 187, in console_main
    code = main()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 164, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 315, in pytest_cmdline_main
    return wrap_session(config, _main)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 268, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 322, in _main
    config.hook.pytest_runtestloop(session=session)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 347, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 111, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 130, in runtestprotocol
    reports.append(call_and_report(item, &quot;call&quot;, log))
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 219, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 258, in call_runtest_hook
    return CallInfo.from_call(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 338, in from_call
    result: Optional[TResult] = func()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 259, in &lt;lambda&gt;
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 166, in pytest_runtest_call
    item.runtest()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 1761, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 192, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File &quot;C:\Users\Yuvraj verma\Desktop\SeleniumTesting\pytestfolder\HomeTest\testofHome.py&quot;, line 62, in test_firstPageCheck
    lg.info(&quot;Sending the gender&quot;+GetData[&quot;gender&quot;])
Message: &#x27;Sending the genderMale&#x27;
Arguments: ()
--- Logging error ---
Traceback (most recent call last):
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 1100, in emit
    msg = self.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 943, in format
    return fmt.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 681, in format
    s = self.formatMessage(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 650, in formatMessage
    return self._style.format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 440, in format
    return self._format(record)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\logging\__init__.py&quot;, line 436, in _format
    return self._fmt % values
ValueError: unsupported format character &#x27;)&#x27; (0x29) at index 51
Call stack:
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\runpy.py&quot;, line 86, in _run_code
    exec(code, run_globals)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\Scripts\py.test.exe\__main__.py&quot;, line 7, in &lt;module&gt;
    sys.exit(console_main())
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 187, in console_main
    code = main()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\config\__init__.py&quot;, line 164, in main
    ret: Union[ExitCode, int] = config.hook.pytest_cmdline_main(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 315, in pytest_cmdline_main
    return wrap_session(config, _main)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 268, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 322, in _main
    config.hook.pytest_runtestloop(session=session)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\main.py&quot;, line 347, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 111, in pytest_runtest_protocol
    runtestprotocol(item, nextitem=nextitem)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 130, in runtestprotocol
    reports.append(call_and_report(item, &quot;call&quot;, log))
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 219, in call_and_report
    call = call_runtest_hook(item, when, **kwds)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 258, in call_runtest_hook
    return CallInfo.from_call(
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 338, in from_call
    result: Optional[TResult] = func()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 259, in &lt;lambda&gt;
    lambda: ihook(item=item, **kwds), when=when, reraise=reraise
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\runner.py&quot;, line 166, in pytest_runtest_call
    item.runtest()
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 1761, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_hooks.py&quot;, line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_manager.py&quot;, line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\pluggy\_callers.py&quot;, line 39, in _multicall
    res = hook_impl.function(*args)
  File &quot;C:\Users\Yuvraj verma\AppData\Local\Programs\Python\Python310\lib\site-packages\_pytest\python.py&quot;, line 192, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File &quot;C:\Users\Yuvraj verma\Desktop\SeleniumTesting\pytestfolder\HomeTest\testofHome.py&quot;, line 62, in test_firstPageCheck
    lg.info(&quot;Sending the gender&quot;+GetData[&quot;gender&quot;])
Message: &#x27;Sending the genderMale&#x27;
Arguments: ()
<br/> -------------------------------Captured log call-------------------------------- <br/>INFO     test_firstPageCheck:testofHome.py:31 Sending the nameTanger
INFO     test_firstPageCheck:testofHome.py:38 Sending the emailTanger@gmail.com
INFO     test_firstPageCheck:testofHome.py:62 Sending the genderMale<br/></div></td></tr></tbody></table></body></html>